### PR TITLE
Removed Duplicate

### PR DESCRIPTION
**Pull request by kulits**
Removed duplicate instance of "Magmar" in default config.

### CODE REVIEW

**Comment by DeadElle** [comment]
Nice catch ;)

File: PoGo.PokeMobBot.Logic/Settings.cs [hunk 1]
```diff
@@ -419,7 +419,6 @@ public class GlobalSettings
                 PokemonId.Scyther,
                 PokemonId.Magmar,
```
Comment (DeadElle) [inline PoGo.PokeMobBot.Logic/Settings.cs]: Nice catch, but following pokemon's IDs, the first one should be removed instead ;)

**Comment by DurtyFree** [comment]
Please consider @DeadElle 's comment @idontcarebear

**Comment by kulits** [comment]
Oops, I missed that.  Here's an updated commit.

**Comment by DurtyFree** [comment]
Thx:)

**Pull request merged**